data/hai missions.txt

```diff
@@ -1061,3 +1061,252 @@ mission "Hai Honeymoon"
+					goto board
+			label fake
+			`	You start making up stories about how beautiful the skies of Earth are. You describe forests teeming with life, and do your best to remember ancient civilizations and their monuments. From the looks of it she believes you, and your lie seems to have sated her curiosity for now. If the day where the Hai are allowed to move around human space ever comes, though, she will no doubt be extremely disappointed.`
+					goto board
+			label truth
+			`	She looks at you in confusion, as if trying to understand what you mean, so you explain it to her. "Centuries of widespread deforestation and pollution badly damaged the ecosystem. If you ever managed to visit, you'd be better off going to museums or monuments, but even then, there are many thieves roaming the streets, and it is very dangerous to be out at night. Lots of folks born there just think about how to get out."`
+			`	You keep telling her about the issues facing humanity's birthplace, and her previous eagerness to learn about the planet fades. "How could your ancestors let their own home become like this?"`
+			choice
+				`	"It's rather complicated, and I'm not a historian, sociologist, or economist, but I can try and explain what I do know."`
+				`	"I can tell you what I do know from watching movies and what I can remember from school."`
+			`	You recite what you read in school books or saw in documentaries, telling her about how many humans since millennia ago refused to take care of the planet, and how that piled up over hundreds of years.`
+			`	Your explanation seems to only torment Nanachi even more. "But why is your capital so full of criminals? Is human space going through a crisis that hit even your capital?"`
+			choice
+				`	"In some sense, the crisis has been going on since we were aware of it."`
+			`	You do your best in telling her of the historical presence of crime on Earth, and of how little was done to address the root of the problem.`
+			`	"I see..." She says. "I just thought it would be fun to hear about another large planet like Hai-home. Sorry for asking about it, Captain, I'm sure having to speak of it wasn't a pleasant feeling."`
+			`	Although you're sure that even in Hai space there is social inequality, she probably can't fathom that the home planet of a species could be in such bad shape, considering the Hai even go as far as to send aid to the Unfettered on a regular basis.`
+					goto board
+			label board
+			`	You two reach the job board and decide to start looking for jobs suited for Nanachi's ship.`
+			choice
+				`	"So, do you mind if I help out?  I know the job board can be confusing the first few times you check it."`
+				`	"Would you like me to accompany you to the job board?"`
```

Comment:
We're already here...
I get the sense that this was added to break so that this wasn't just a single choice. How about instead you say `"Do you think you have this on your own?"` for a second choice, then have Nanachi say `"Yeah, I should be good."`